app/models/user.rb

```diff
@@ -267,4 +268,12 @@ def yank_gems
       deletions.create(version: v)
     end
   end
+
+  def toxic_email_domain
+    domain             = email.split("@").last
+    toxic_domains_path = Pathname.new(Gemcutter::Application.config.toxic_domains_filepath)
+    toxic = toxic_domains_path.exist? && toxic_domains_path.readlines.grep(/^#{domain}$/).any?
```

Comment:
I worry about the impact to performance by needing to searching through an ever growing list of domains. Have you run any benchmark tests to see what the cost is? Is there an alternative approach to solve this problem that doesn't have the same performance cost?